mlflow/server/js/src/experiment-tracking/components/ExperimentListView.js

```diff
@@ -231,7 +231,8 @@ export class ExperimentListView extends Component {
 const classNames = {
   experimentListOuterContainer: {
     boxSizing: 'border-box',
-    marginLeft: '64px',
+    marginLeft: '24px',
+    marginRight: '8px',
```

Comment:
The runs table is wrapped by `PageContainer` which has a 16px left padding.

<img width="1058" alt="image" src="https://user-images.githubusercontent.com/17039389/200224447-2e61fcf2-7c72-466a-bd8e-8efaa207f8bc.png">